### DISCUSSION TITLE
Toonsquid auto resizing imported images

### DISCUSSION THREAD

**Discussion by 8-8Sophie8-8** [keiwando/toonsquid#627] [Q&A] [answered]
Hello to TOONSQUID community, I keep having this problem with Toonsquid auto resizing images and have no clue what is causing this issue .

**Example**: let’s say I work on a drawing in 1920x1080 canva size;
Then I enlarge that drawing to make sure that it fills up the whole canvas.
Then I opened Toonsquid and create a 1920x1080 canva size, when I import that  drawing  into Toonsquid, the drawing got auto resize, even when I imported it into the same canva size that I use in the drawing program, it did not retain it’s original size .

My question: does anyone know how to stop drawing from auto resizing when I import in Toonsquid.

**Comment by keiwando** [comment, marked as answer]
Hello!
If you are importing an image with the same resolution as the project, it should fill the entire canvas and that's what happens when I try it on my iPad. Could you maybe share an example image file here and a short [screen recording](https://support.apple.com/en-us/102653) that shows what exactly happens when you try to import that image into a new project with the same resolution?

**Reply by 8-8Sophie8-8** [reply]
no problem with this , so am not really sure what was causing it , after my iPad got new system upgrade, I had no problems with this issue.
